src/test/kotlin/dev/stalla/builder/validating/episode/ValidatingEpisodePodcastindexSeasonBuilderTest.kt

```diff
@@ -0,0 +1,84 @@
+package dev.stalla.builder.validating.episode
+
+import assertk.all
+import assertk.assertAll
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFalse
+import assertk.assertions.isNotNull
+import assertk.assertions.isNull
+import assertk.assertions.isTrue
+import assertk.assertions.prop
+import dev.stalla.builder.episode.EpisodePodcastindexSeasonBuilder
+import dev.stalla.model.anEpisodePodcastindexSeason
+import dev.stalla.model.podcastindex.PodcastindexSeason
+import org.junit.jupiter.api.Test
+
+internal class ValidatingEpisodePodcastindexSeasonBuilderTest {
+
+    @Test
+    internal fun `should not build an Episode Podcastindex Season with when all the fields are missing`() {
```

Comment:
```suggestion
    internal fun `should not build an Episode Podcastindex Season when all the mandatory fields are missing`() {
```

Also redundant, there is only one mandatory field